src/cli/cli.ts

```diff
@@ -44,6 +44,7 @@ const registriesByName: {[name: string]: (kit: ContractKit) => Registry} = {
 	"moola-v2":    mainnetRegistryMoolaV2,
 	"savingscelo": mainnetRegistrySavingsCELO,
 	"celodex":     mainnetRegistryCeloDex,
+	"symmetric":   mainnetRegistrySymmetric
```

Comment:
i think there's a SymmetricV2 based on BalancerV2 coming soon too, maybe need more work when the v2 is deployed to Celo